FBC/FunctionalEntities/NorthAmericanEntities/USExampleIndividuals.rdf

```diff
@@ -137,13 +140,141 @@
+		<fibo-fbc-fct-ra:isRegisteredIn rdf:resource="&fibo-fbc-fct-usjrga;OhioBusinessFilingPortal"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;TheProctorAndGambleCompanyBusinessEntityIdentifier">
+		<fibo-fbc-fct-breg:hasHeadquartersAddress rdf:resource="&fibo-fbc-fct-usind;TheProctorAndGambleCompanyCorporateAddress"/>
+		<fibo-fbc-fct-breg:hasWebsite rdf:datatype="&xsd;anyURI">https://us.pg.com/</fibo-fbc-fct-breg:hasWebsite>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fbc-fct-usind;AlphabetIncCorporateAddress">
+		<rdf:type rdf:resource="&fibo-fbc-fct-breg;RegistrationAddress"/>
+		<rdfs:label>Alphabet Inc. headquarters  / registered address</rdfs:label>
+		<skos:definition>individual representing the registered address for Alphabet Inc.</skos:definition>
+		<fibo-fbc-fct-breg:hasAddressLine1>1600 Amphitheatre Parkway</fibo-fbc-fct-breg:hasAddressLine1>
+		<fibo-fbc-fct-breg:hasCity>Mountain View</fibo-fbc-fct-breg:hasCity>
+		<fibo-fbc-fct-breg:hasCountry rdf:resource="&lcc-3166-1;UnitedStates"/>
+		<fibo-fbc-fct-breg:hasPostalCode>94043</fibo-fbc-fct-breg:hasPostalCode>
+		<fibo-fbc-fct-breg:hasSubdivision rdf:resource="&lcc-3166-2-us;California"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fbc-fct-usind;AlphabetIncLegalEntityIdentifierRegistryEntry">
+		<rdf:type rdf:resource="&fibo-fbc-fct-breg;LegalEntityIdentifierRegistryEntry"/>
+		<rdfs:label>Alphabet Inc. legal entity identifier registry entry</rdfs:label>
+		<skos:definition>individual representing the legal entity identifier Global LEI Index registry entry for Alphabet Inc.</skos:definition>
+		<fibo-fbc-fct-breg:hasInitialRegistrationDate rdf:datatype="&xsd;dateTime">2015-08-31T09:16:00-07:00</fibo-fbc-fct-breg:hasInitialRegistrationDate>
+		<fibo-fbc-fct-breg:hasRegistrationStatus rdf:resource="&fibo-fbc-fct-breg;IssuedStatus"/>
+		<fibo-fbc-fct-breg:hasRegistrationStatusRevisionDate rdf:datatype="&xsd;dateTime">2015-08-31T09:16:00-07:00</fibo-fbc-fct-breg:hasRegistrationStatusRevisionDate>
+		<fibo-fbc-fct-breg:hasRenewalDate rdf:datatype="&xsd;dateTime">2021-01-09T10:59:00-08:00</fibo-fbc-fct-breg:hasRenewalDate>
+		<fibo-fbc-fct-breg:hasValidationLevel rdf:resource="&fibo-fbc-fct-breg;EntityValidationLevelFullyCorroborated"/>
+		<fibo-fnd-rel-rel:comprises rdf:resource="https://www.gleif.org/lei/5493006MHB84DD0ZWV18"/>
+		<fibo-fnd-rel-rel:isPartOf rdf:resource="&fibo-fbc-fct-usfsind;GlobalMarketsEntityIdentifierRegistry"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fbc-fct-usind;AppleIncCorporateAddress">
+		<rdf:type rdf:resource="&fibo-fbc-fct-breg;RegistrationAddress"/>
+		<rdfs:label>Apple Inc. headquarters  / registered address</rdfs:label>
+		<skos:definition>individual representing the registered address for Apple Inc.</skos:definition>
+		<fibo-fbc-fct-breg:hasAddressLine1>One Apple Park WY</fibo-fbc-fct-breg:hasAddressLine1>
```

Comment:
WY is a variant spelling, and was used in a filing that I took this from.  I just checked the GLEIF site and it is spelled that way in the GLEIF Beta 2 site, which I believe is where I grabbed it.

Reply:
I agree Wy is an OK abbreviation like Rd, the question is whether it should be all caps.
In GLEIF the whole address is capitalized ONE APPLE PARK WY so that's not a good indication. Given that you had the rest of the address mixed case then this should be too. But not worth pressing so I'll mark this as resolved.